client/src/api/ssotRetrieval.js

```diff
@@ -112,12 +111,31 @@ const deleteAttributesForActivities = (robotId, activityIdList) => {
   });
 };
 
+/**
+ * @description Overwrites an existing sssot in the backend with a new one
+ * @param {String} robotId Id of the robot that is being overwritten
+ * @param {String} ssot New ssot to be written to the database
+ */
+const updateRobot = async (robotId, ssot) => {
```

Comment:
Not sure about if we should have this filename stay the same. retrieval suggests getting something, while for example with this function we are overwriting.
I am not saying this file should not be placed here (because I think the logical grouping of all functions here makes sense), but rather that the file should be renamed.

Reply:
I would rather not extend files, but the name idea sounds good in general